doc/user-getting-started-c.md

```diff
@@ -19,9 +19,8 @@ Save it to a folder and call it `helloworld.c`.
 
 ## Build the program
 
-Compile `helloworld.c` with `myst-gcc`, and place it under a subfolder
-`appdir`. `myst-gcc` is a wrapper for musl-gcc which compiles C programs
-dynamically linked to MUSL libc instead of the default glibc.
+Compile `helloworld.c` with `gcc`, and place it under a subfolder
```

Comment:
Nit: Unix/Linux developers usually refer to "subdirectories" rather than "subfolders". I think "subfolders" is a Windows term.